Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -1919,7 +1919,7 @@ void MainWindow::Disconnect()
 
 void MainWindow::login()
 {
-    QString nick = ttSettings->value(SETTINGS_GENERAL_NICKNAME, QCoreApplication::translate("MainWindow", SETTINGS_GENERAL_NICKNAME_DEFAULT)).toString();

```

Comment:
Keep SETTINGS_GENERAL_NICKNAME_DEFAULT but set SETTINGS_GENERAL_NICKNAME_DEFAULT to an empty string

Reply:
Move "QT_TRANSLATE_NOOP("MainWindow", "NoName")" from SETTINGS_GENERAL_NICKNAME_DEFAULT to a new macro called DEFAULT_NICKNAME in common.cpp.  The DEFAULT_NICKNAME should then be used in getDisplayName().